ts/features/wallet/bancomat/utils/bancomatInformationBottomSheet.tsx

```diff
@@ -25,34 +22,32 @@ const styles = StyleSheet.create({
  * This will be also visualized inside a bottomsheet after an addition of a new bancomat
  */
 export default (onAdd?: () => void) => {
-  const { present: openBottomSheet, dismiss } = useIOBottomSheetRaw(
-    385,
-    bottomSheetContent
+  const { present, bottomSheet, dismiss } = useIOBottomSheet(
```

Comment:
This is used in `BancomatInformation` component, but we need also to add the rendering of the returned bottomsheet, in order to display it. Here my proposal:

```typescript
const BancomatInformation: React.FunctionComponent<Props> = props => {
  const { present, bottomSheet } = bancomatInformationBottomSheet(
    props.onAddPaymentMethod
  );
  return (
    <View testID={"bancomatInformation"}>
      {bottomSheet}
      <View style={styles.titleContainer}>
      ....
```

Reply:
Same as above some BS escaped from the refactor 😞   thanks for noticing!